<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>2321312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>钢琴陪练系统</string>
  </property>
  <widget class="QWidget" name="_pLeftMenuWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>206</width>
     <height>380</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>206</width>
     <height>380</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="_pLeftMenuBtn_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>146</y>
      <width>170</width>
      <height>52</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>52</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_pLeftMenuBtn_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>170</width>
      <height>52</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>170</width>
      <height>52</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="MainWidgetTitle" name="_pCaptionWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>1200</width>
     <height>75</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>75</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="CapBtn" name="_pCapBtn_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>153</width>
      <height>52</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>153</width>
      <height>52</height>
     </size>
    </property>
    <property name="text">
     <string>乐谱自学</string>
    </property>
   </widget>
   <widget class="CapBtn" name="_pCapBtn_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>153</width>
      <height>52</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>153</width>
      <height>52</height>
     </size>
    </property>
    <property name="text">
     <string>纠错陪练</string>
    </property>
   </widget>
   <widget class="CapBtn" name="_pCapBtn_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>153</width>
      <height>52</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>153</width>
      <height>52</height>
     </size>
    </property>
    <property name="text">
     <string>在线教学</string>
    </property>
   </widget>
   <widget class="CapBtn" name="_pCapBtn_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>153</width>
      <height>52</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>153</width>
      <height>52</height>
     </size>
    </property>
    <property name="text">
     <string>考试系统</string>
    </property>
   </widget>
   <widget class="CapBtn" name="_pCapBtn_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>153</width>
      <height>52</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>153</width>
      <height>52</height>
     </size>
    </property>
    <property name="text">
     <string>老师点评</string>
    </property>
   </widget>
   <widget class="QWidget" name="_pCaptionWidget2" native="true">
    <property name="geometry">
     <rect>
      <x>899</x>
      <y>0</y>
      <width>241</width>
      <height>75</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>241</width>
      <height>75</height>
     </size>
    </property>
    <widget class="QLabel" name="_pHeadLab">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>54</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>54</width>
       <height>51</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="_pHeadInfoLab">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>131</width>
       <height>75</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>75</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="_pCloseBtn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="_pSmallBtn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>40</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="_pBottomLab">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>660</y>
     <width>1198</width>
     <height>34</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1198</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string>所有内容由武汉巴赫风文化传媒倾心打造，版权归巴赫风所有</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CapBtn</class>
   <extends>QPushButton</extends>
   <header>CapBtn.h</header>
  </customwidget>
  <customwidget>
   <class>MainWidgetTitle</class>
   <extends>QWidget</extends>
   <header>MainWidgetTitle.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
